frontend/src/views/SubmissionsView.vue

```diff
@@ -256,6 +256,20 @@ onMounted(async () => {
           />
         </template>
       </Column>
+      <Column
+        field="aaiUpdated"
+        header="Authorization and Approvals Insight (AAI) updated"
```

Comment:
hopefully Tyler or someone can go through and improve the visual design of some of these things. thats a very wide column and long header text for just a checkbox. lol